### ISSUE TITLE
More privesc scripts

### ISSUE THREAD

**Issue by zAbuQasem** [brightio/penelope#8]
It would be awesome if you added deepce (Docker enumeration ) , PrivescCheck (Fancy script)
```python3
options.recon_scripts = {
'Unix':[
	'https://github.com/carlospolop/PEASS-ng/releases/latest/download/linpeas.sh',
	'https://raw.githubusercontent.com/diego-treitos/linux-smart-enumeration/master/lse.sh',
	'https://raw.githubusercontent.com/stealthcopter/deepce/main/deepce.sh'
],
'Windows':[
	'https://raw.githubusercontent.com/PowerShellEmpire/PowerTools/master/PowerUp/PowerUp.ps1',
	'https://raw.githubusercontent.com/itm4n/PrivescCheck/master/PrivescCheck.ps1'
]}
```

**Comment by brightio** [comment]
I am planning to make that customizable with batch files so you can specify your personal batch file with -b and you will be able to upload your favorite scripts and also perform automated actions on the targets.

**Comment by brightio** [comment]
Hi @zAbuQasem, this is from the updated README:

> The batch command by default uploads predefined privilege escalation scripts. You can add more scripts or modify this and other default behaviours by using a configuration file (See extras/penelope.conf). This file can be speficied with -r in the command line or be placed in ~/.penelope/penelope.conf

Please try it when you have time and tell me if ti works for you. Thank you!

**Comment by brightio** [comment]
@zAbuQasem the 2 scripts you proposed are now included by default (commit 7db5174). For custom modules and options you can use the ~/.penelope/peneloperc file.
